task-tracker-webapp/src/main/java/com/isa/tasktrackerwebapp/model/Task.java

```diff
@@ -1,11 +1,21 @@
 package com.isa.tasktrackerwebapp.model;
 
+import jakarta.validation.constraints.FutureOrPresent;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+
 import java.time.LocalDate;
 
 public class Task {
+    @NotBlank(message = "Nazwa zadania nie może być pusta.")
     private String taskName;
+    @FutureOrPresent(message = "Nie można wprowadzić daty z przeszłości.")
+    @NotNull(message = "Data rozpoczęcia nie może być pusta.")
     private LocalDate taskStart;
+    @FutureOrPresent(message = "Nie można wprowadzić daty z przeszłości.")
+    @NotNull(message = "Data zakończenia nie może być pusta.")
     private LocalDate taskEnd;
+    @NotBlank(message = "Opis zadania nie moży być pusty.")
     private String taskDescription;
     private User user;
```

Comment:
a tutaj walidacji nie ma? task, może nie mieć usera? czy on jest dodawany w innym momencie?

Reply:
on jest dodawany automatycznie - to zawsze będzie zalogowany użytkownik (przynajmniej w tym sprincie). Czy w tej sytuacji dodawać walidację?

Reply:
pomyślimy może o tym w kolejnym sprincie...